dockerfiles/ci/bookworm/.env

```diff
@@ -0,0 +1,2 @@
+BOOKWORM_CURRENT_VERSION=1
+BOOKWORM_NEXT_VERSION=2
```

Comment:
I do not understand the point of this file?

Reply:
The current buster builds are "unversioned." This can cause issues (like the one you mention below with nts-asan failing). This file is a way to version the build, although it's not plumbed all the way through everything like the CI and such. We do this in the libdatadog images and it's worked nicely there.